infra/aws/terraform/prow-build-cluster/eks.tf

```diff
@@ -44,6 +44,8 @@ module "eks" {
     },
   ]
 
+  iam_role_permissions_boundary = "arn:aws:iam::${local.account_id}:policy/ProvisionerPermissionBoundary"
```

Comment:
Is it possible to automatically figure out this ARN with data resource? Doing that will also ensure that this permission boundary exists.

Reply:
Yeah, that's a great idea